File: root_VS2010_TemplateBase/programs/C#/Samples/WinAzure_sample/ProjectX_sample/Aspx/Common/sampleControl.ascx.cs
using System;
using System.Data;
using System.Configuration;
using System.Collections;
using System.Web;
using System.Web.Security;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;
using System.Web.UI.HtmlControls;

using Touryo.Infrastructure.Framework.Presentation;

public partial class Aspx_Common_WebUserControl : System.Web.UI.UserControl
{
    /// <summary>ユーザコントロールにイベントハンドラを実装可能にしたのでそのテスト。</summary>
    /// <param name="fxEventArgs">イベントハンドラの共通引数</param>
    /// <returns>URL</returns>
    protected string UOC_btnUCButton_Click(FxEventArgs fxEventArgs)
    {
        Response.Write("UOC_btnUCButton_Clickを実行できた。");

        return "";
    }
}

File: root_VS2012_TemplateBase/programs/VB/Samples/WebApp_sample/ProjectX_sample/Aspx/Common/sampleControl.ascx.vb
Imports System
Imports System.Data
Imports System.Configuration
Imports System.Collections
Imports System.Web
Imports System.Web.Security
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports System.Web.UI.WebControls.WebParts
Imports System.Web.UI.HtmlControls

Imports Touryo.Infrastructure.Framework.Presentation

Public Partial Class Aspx_Common_WebUserControl
	Inherits System.Web.UI.UserControl
	''' <summary>ユーザコントロールにイベントハンドラを実装可能にしたのでそのテスト。</summary>
	''' <param name="fxEventArgs">イベントハンドラの共通引数</param>
	''' <returns>URL</returns>
	Protected Function UOC_btnUCButton_Click(fxEventArgs As FxEventArgs) As String
		Response.Write("UOC_btnUCButton_Clickを実行できた。")

		Return ""
	End Function
End Class
